### PR TITLE
SK-1482-WEB-Emails-Exchange-manage_billing

### PR DESCRIPTION
**Pull request by BrenKt**

### CODE REVIEW

File: pages/web_cloud/email_and_collaborative_solutions/microsoft_exchange/manage_billing_exchange/guide.fr-fr.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: 'Gérer la facturation de vos comptes Exchange'
 excerpt: 'Découvrez comment gérer la facturation de votre service Exchange'
-updated: 2020-04-20
+updated: 2020-08-08
```
Comment (Y0Coss) [inline pages/web_cloud/email_and_collaborative_solutions/microsoft_exchange/manage_billing_exchange/guide.fr-fr.md]:
```suggestion
updated: 2024-08-08
```

File: pages/web_cloud/email_and_collaborative_solutions/microsoft_exchange/manage_billing_exchange/guide.fr-fr.md [hunk 2]
```diff
@@ -71,6 +71,10 @@ Dirigez-vous dans l'onglet `Comptes e-mail`{.action}, cliquez sur le bouton `Act
 
 #### Supprimer des comptes
 
+> [!warning]
+>
+> Pour pouvoir supprimer un compte e-mail sur une plateforme Private Exchange, **il est nécessaire que celui-ci est été créé** et attaché à un nom de domaine.
```
Comment (Y0Coss) [inline pages/web_cloud/email_and_collaborative_solutions/microsoft_exchange/manage_billing_exchange/guide.fr-fr.md]:
```suggestion
> Pour pouvoir supprimer un compte e-mail sur une plateforme Private Exchange, **il est nécessaire que celui-ci ait été créé** et attaché à un nom de domaine.
```

**Review by bs-ovh — APPROVED** [review]

File: pages/web_cloud/email_and_collaborative_solutions/microsoft_exchange/manage_billing_exchange/guide.es-es.md [hunk 3]
```diff
@@ -69,6 +69,10 @@ Acceda a la pestaña `Cuentas de correo`{.action}, haga clic en el botón `Accio
 
 #### Eliminar cuentas
 
+> [!warning]
+>
+> Para poder eliminar una cuenta de correo en una plataforma Private Exchange, es necesario que esta haya sido creada** (asociada al dominio de su plataforma Exchange). No se puede eliminar una cuenta no configurada.
```
Comment (Y0Coss) [inline pages/web_cloud/email_and_collaborative_solutions/microsoft_exchange/manage_billing_exchange/guide.es-es.md]:
```suggestion
> Para poder eliminar una cuenta de correo en una plataforma Private Exchange, **es necesario que esta haya sido creada** (asociada al dominio de su plataforma Exchange). No se puede eliminar una cuenta no configurada.
```

**Review by Y0Coss — COMMENTED** [review]
Dates need to be updated

**Pull request merged**